### PR TITLE
Update extensionProperties to single array of structs

### PR DESCRIPTION
**Pull request by jwforres**

### CODE REVIEW

**Comment by jwforres** [comment]
@liggitt yeah looks like jsstr isnt a supported filter in html/template either

File: pkg/assets/handlers.go [hunk 1]
```diff
@@ -155,20 +154,20 @@ type WebConsoleVersion struct {
...
+var extensionPropertiesTemplate = template.Must(template.New("webConsoleExtensionProperties").Parse(`
 window.OPENSHIFT_EXTENSION_PROPERTIES = {
-  {{ $values := .ExtensionPropertyValues }}
-  {{ range $i, $key := .ExtensionPropertyKeys }}
-  {{ if $i }},{{ end }}
-  {{ $value := index $values $i }}
-  "{{ $key }}": "{{ $value }}"
-  {{ end }}
+{{ range $i, $property := .ExtensionProperties }}{{ if $i }},{{ end }}
+  "{{ $property.Key | jsstr }}": "{{ $property.Value | jsstr }}"{{ end }}
```
Comment (liggitt) [inline pkg/assets/handlers.go]: `| js` seems to do what we want. It was the additional inferred filters in the html/template package that was making the output weird

**Comment by jwforres** [comment]
@liggitt updated

**Comment by jwforres** [comment]
hold.. left in a debug log statement

**Comment by jwforres** [comment]
@liggitt ok should be good now

File: pkg/cmd/server/origin/asset.go [hunk 2]
```diff
@@ -144,14 +143,15 @@ func (c *AssetConfig) buildAssetHandler() (http.Handler, error) {
...
-	extensionKeys := []template.JSStr{}
-	extensionValues := []template.JSStr{}
+func extensionPropertyArray(extensionProperties map[string]string) []assets.WebConsoleExtensionProperty {
+	extensionPropsArray := []assets.WebConsoleExtensionProperty{}
 	for key, value := range extensionProperties {
-		extensionKeys = append(extensionKeys, template.JSStr(key))
-		extensionValues = append(extensionValues, template.JSStr(value))
+		extensionPropsArray = append(extensionPropsArray, assets.WebConsoleExtensionProperty{
+      Key: key,
+      Value: value,
```
Comment (liggitt) [inline pkg/cmd/server/origin/asset.go]: gofmt not complain about indent here?

**Comment by liggitt** [comment]
```
!!! gofmt needs to be run on the following files: 

./pkg/assets/handlers.go

./pkg/cmd/server/origin/asset.go
```

**Comment by liggitt** [comment]
fix gofmt, then LGTM

**Comment by jwforres** [comment]
gofmt should be fixed now

**Comment by jwforres** [comment]
[merge]

**Pull request merged**